lib/ai/tools/web-search.ts

```
@@ -148,7 +150,13 @@ async function makePerplexityRequest(payload: any, abortSignal: AbortSignal) {
 
   if (!response.ok) {
     const responseBody = await getResponseBody(response);
-    const errorData = JSON.parse(responseBody);
+    let errorData;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)